e2e/libs/sharemanager/crd.py

```
@@ -16,3 +20,49 @@ def list(self, label_selector=None):
+        for i in range(self.retry_count):
+            sharemanager = self.get(name)
+            current_status = sharemanager["status"]["state"]
+            logging(f"wait sharemanager {name} running, current status = {current_status}")
+            if current_status == "running":
+                return
+            time.sleep(self.retry_interval)
+
+        assert False, f"Failed to wait sharemanager {name} in running state"
+
+    def wait_for_restart(self, name, last_creation_time):
+        for i in range(self.retry_count):
```

Ruff

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)